### PR TITLE
Feature/pelagos 4183 modernize stats page

### PR DESCRIPTION
**Pull request by mickel1138**

### CODE REVIEW

File: assets/js/stats.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+var overviewSections;
+var style = getComputedStyle(document.body);
+var theme = {};
+
+theme.main = style.getPropertyValue('--color-main');
+theme.secondary = style.getPropertyValue('--color-menu');
+theme.dark = style.getPropertyValue('--color-headerMiddle');
+theme.light = style.getPropertyValue('--color-headerTop');
+
+$(document).ready(function() {
```
Comment (praneethpr) [inline assets/js/stats.js:18]: Because its a rewrite should we use regular javascript instead of jQuery?

File: assets/css/stats.css [hunk 2]
```diff
@@ -0,0 +1,142 @@
+#stats-tabs {
```
Comment (praneethpr) [inline assets/css/stats.css:1]: Should this file be a git move?

File: assets/js/stats.js (hunk 1 above)
Comment (mickel1138) [reply assets/js/stats.js:18]: These section is going away in the next ticket.

File: assets/js/stats.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
+const $ = require('jquery');
+global.$ = global.jQuery = $;
```
[the remark this answers is not in the document]
Comment (mickel1138) [reply assets/js/stats.js:2]: I still not sure how webpack manages globals. I think we should have used "app" as a common app in the base template.

File: assets/css/stats.css (hunk 2 above)
Comment (mickel1138) [reply assets/css/stats.css:1]: There is a minor change, but I didn't want to odd thing to happen again. I can delete the other one.

Comment (praneethpr) [reply assets/css/stats.css:1]: If there are no changes in the file, i think git move should work just fine

File: assets/js/stats.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (praneethpr) [reply assets/js/stats.js:2]: Because we use $ in the static/js/ files we need the global, because client side javascript doesn't recognize "require"

**Pull request closed without merging**